src/components/Book.jsx

```diff
@@ -1,12 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { CircularProgressbar } from 'react-circular-progressbar';
 import PropTypes from 'prop-types';
 import { useDispatch } from 'react-redux';
 import { removeBook, getBooks } from '../redux/books/booksSlice';
+import 'react-circular-progressbar/dist/styles.css';
 
 // eslint-disable-next-line camelcase
```

Comment:
- [ ] Please avoid the use of `eslint` disable property because it is against the general requirement of the project, Eslinter helps to write better code and by disabling it you are not guaranteed to provide the quality code. Kindly solve the issue manually.